kernel/syno-armada38x-5.2/Makefile

```diff
@@ -0,0 +1,21 @@
+ARCH ?= armada38x
+TCVERSION ?= 5.2
+PKG_NAME = armada38x
+PKG_BRANCH = 5644
+PKG_ARCH = armada38x
```

Comment:
Probably makes sense to use `$(PKG_NAME)` here. We already repeat it more than I'd like ;)